### PR TITLE
Drone set CORE_BRANCH to stable21

### PR DESCRIPTION
**Pull request by PVince81**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -137,7 +137,7 @@ steps:
     image: nextcloudci/php7.3:php7.3-5
     environment:
       APP_NAME: spreed
-      CORE_BRANCH: master
+      CORE_BRANCH: stable21
       GUESTS_BRANCH: master
```
Comment (danxuliu) [inline .drone.yml:141]: It looks like `GUESTS_BRANCH` needs to be changed too :-)

**Comment by nickvergessen** [comment]
How did that not match the string replace from #5067

**Comment by PVince81** [comment]
> How did that not match the string replace from #5067 is there a double space or something?

No idea... I did all the replaces manually just to be sure.

File: .drone.yml (hunk 1 above)
Comment (PVince81) [reply .drone.yml:141]: Not needed, because guests app doesn't have such branches.

See https://github.com/nextcloud/guests/pull/543

Comment (danxuliu) [reply .drone.yml:141]: :facepalm: :+1:

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**